apps/src/templates/sectionsRefresh/SectionsSetUp.jsx

```diff
@@ -0,0 +1,56 @@
+import React, {useState} from 'react';
+import i18n from '@cdo/locale';
+import SetUpSectionForm from './SetUpSectionForm';
+import Button from '@cdo/apps/templates/Button';
+import moduleStyles from './sections-refresh.module.scss';
+
+const useSections = () => {
+  const [sections, setSections] = useState([{}]);
+
+  const updateSection = (sectionIdx, keyToUpdate, val) => {
+    const newSections = sections.map((section, idx) => {
+      if (idx === sectionIdx) {
+        return {
+          ...section,
+          [keyToUpdate]: val
+        };
+      } else {
+        return section;
+      }
+    });
+    setSections(newSections);
+  };
+
+  return [sections, updateSection];
+};
+
+export default function SectionsSetUp() {
```

Comment:
are there any conventions for component names? `SetUpSectionsForm` is imperative, while `SectionsSetUp` is not. i don't have strong opinions here, but consistency does make it easier to find things when you're looking for a particular component.